src/app/[practiceCode]/book/page.tsx

```diff
@@ -9,6 +9,54 @@ import { redirect } from 'next/navigation';
 
 const supabase = createServerComponentClient<Database>({ cookies });
 
+export async function generateMetadata({
```

Comment:
Seeing this async function defined here, but is it being called anywhere?

Reply:
the generateMetadata function needed to fetch metadata that requires dynamic values.
https://nextjs.org/docs/app/building-your-application/optimizing/metadata

Reply:
Great job @NuriyaAkh !